nightwatch.js

```diff
@@ -12,8 +12,10 @@ const TRAVIS_JOB_NUMBER = process.env.TRAVIS_JOB_NUMBER;
 
 module.exports = {
   "src_folders": ["test/e2e/specs"],
+  "globals_path": "globals.js",
```

Comment:
Do you actually need a separate file for 2 globals?
Just include them in the config.